### PR TITLE
Multiple fixes for EN and FI language strings

### PR DESCRIPTION
**Pull request by markoaamunkajo**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -835,15 +833,15 @@ <h2>
 							</div>
 							<label class="control control-radio">
 								<span lang="en">Bearer authentication</span>
-								<span lang="fi">Bearer-token</span>
+								<span lang="fi">Todennus haltijatunnuksella</span>
```
Comment (ojousima) [inline src/index.html:836]: I would use "Bearer-token" here in Finnish. It's not really Finnish, but I understand what a "Bearer-token" is where's "haltijatunnus" sounds something out of J.R.R. Tolkien book.

File: src/index.html [hunk 2]
```diff
@@ -1340,7 +1338,7 @@ <h2>
 						<div id="settings_lan_auth-api_key" class="indent hidden">
 							<label for="settings_lan_auth-api_key">
 								<span lang="en">API key (bearer token):</span>
-								<span lang="fi">API-avain (bearer token):</span>
+								<span lang="fi">API-avain (haltijatunnus):</span>
```
Comment (ojousima) [inline src/index.html:1341]: See above comment about bearer token in Finnish

File: src/index.html [hunk 3]
```diff
@@ -1365,7 +1363,7 @@ <h2>
 						<div id="settings_lan_auth-api_key_rw" class="indent hidden">
 							<label for="settings_lan_auth-api_key_rw">
 								<span lang="en">API key (bearer token):</span>
-								<span lang="fi">API-avain (bearer token):</span>
+								<span lang="fi">API-avain (haltijatunnus):</span>
```
Comment (ojousima) [inline src/index.html:1366]: See above comment about bearer token in Finnish

File: src/index.html [hunk 4]
```diff
@@ -131,10 +131,10 @@ <h2>
 				<div class="font-bold">
 					<span lang="en">
 						You're accessing the Ruuvi Gateway configuration wizard from your local area network.
-						For this reason, you cannot adjust the connection type.
+						You cannot adjust the connection type at this time.
 						In order to change the connection type, please reconfigure the Gateway using its Wi-Fi hotspot.
-						To start the hotspot, take a paper clip and press the pinhole button for a
-						short time, found under the power socket.
+						To restart the Wi-Fi hotspot, locate pinhole button under the power socket and press the button
```
Comment (ojousima) [inline src/index.html:136]: `locate pinhole button` -> `locate the pinhole button`?

**Review by ojousima — CHANGES_REQUESTED** [review]

**Review by ojousima — APPROVED** [review]
Thanks

**Review by DG12 — COMMENTED** [review]
Maybe instead of "fixed" "wired" might be better.

**Review by DG12 — COMMENTED** [review]
98) Ruuvi Gateway requires an Internet connection

 unnecessary:  in order to work correctly. ??
++++++++++++++++++
137) press the button **down** briefly                       
681)
702)
2125)
2147)


SB  **in** ?

+++++++++++++=
2124)
If you misconfigured,  ((Awkward))

Better??: To change the configuration

**Pull request merged**